docs/config.md

```diff
@@ -141,6 +141,10 @@ aws {
 }
 ```
 
+:::{tip}
+This scope can also be used for non-Amazon S3 methods such as Ceph (often used with Openstack clouds)
```

Comment:
```suggestion
This scope can also be used to configure access to S3-compatible storage outside of AWS.
```

Reply:
@bentsherman I actually suggested to @jfy133 that he specifically include the term "openstack" so that if people search the docs for it, it'll come up..

Reply:
Yes... I originally tried searching for 'openstack' and 'ceph' in this case, but didn't find anything. It's very misleading the whole scope is called `aws` as I thought that was restricted to just Amazon hosted infrastructure.

Reply:
I removed the openstack mention because I don't want to give the impression that you can use the `aws` scope for other clouds, only S3-compatible storage. So you could access the S3-compatible storage in your openstack instance but not the compute resources, right?

I think it would be fine to mention examples of S3-compatible storage like Ceph and MinIO.

Reply:
Fair enough! Mentioning Ceph and MinIO would be sufficient for me if you're OK with it :) I did try searching `ceph`, and `minIO` is another storage/api option in the OpenStack instance I'm on ([de.NBI cloud](https://cloud.denbi.de/wiki/quickstart/))  so even having that would have worked.

It would require a much larger change but I wonder if the `aws` scope could be split/partially duplcated with an `s3` scope instead (But like I said, I'm naïve with all this stuff so I don't know if that would be valid either).

Reply:
We definitely should try to communicate better how to use Nextflow with S3-compatible storage, maybe add a section to the page on Amazon S3. For this PR I will just add the examples to the tip...

Reply:
Sounds good! Would you like me to make an issue for more generalised docs?

Reply:
Actually, we might as well do it in this PR... can you add a brief section to the S3 page, showing how you configure S3-compatible storage? You could use your OpenStack / Ceph setup as an example. Basically just show the necessary configuration, I can help with the prose.

Reply:
Done!